node/public.go

```diff
@@ -0,0 +1,66 @@
+// Package node provides a high level wrapper around tendermint services.
+package node
+
+import (
+	"fmt"
+
+	"github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/libs/log"
+	"github.com/tendermint/tendermint/libs/service"
+	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/privval"
+	"github.com/tendermint/tendermint/proxy"
+	"github.com/tendermint/tendermint/types"
+)
+
+// NewDefault constructs a tendermint node service for use in go process that
+// host their own process-local tendermint node. This is roughly
+// equivalent to running tendermint in it's own process communicating
+// to an external ABCI application.
+func NewDefault(conf *config.Config, logger log.Logger) (service.Service, error) {
+	return newDefaultNode(conf, logger)
+}
+
+// New constructs a tendermint node for use in an integrated
```

Comment:
What does `integrated` mean here?

Also, `The ClientCreator makes it possible to construct.`, what does that mean?

Reply:
I've edited the comment in the next commit.